tests/unit/IOUUtilsTest.js

```diff
@@ -0,0 +1,139 @@
+import * as IOUUtils from '../../src/libs/IOUUtils';
+import * as ReportUtils from '../../src/libs/ReportUtils';
+
+let iouReport;
+let reportActions;
+const ownerEmail = 'owner@iou.com';
+const managerEmail = 'manager@iou.com';
+
+function createIOUReportAction(type, amount, currency, {IOUTransactionID, isOnline} = {}) {
+    let moneyRequestAction = ReportUtils.buildOptimisticIOUReportAction(
+        1,
+        type,
+        amount,
+        currency,
+        'Test comment',
+        [managerEmail],
+        '',
+        IOUTransactionID,
+        iouReport.reportID,
+    );
+
+    // Default is to create requests offline, if this is specified then we need to remove the pendingAction
+    if (isOnline) {
+        moneyRequestAction = {
+            ...moneyRequestAction,
+            pendingAction: null,
+        };
+    }
+    reportActions.push(moneyRequestAction);
+    return moneyRequestAction;
+}
+
+function cancelMoneyRequest(moneyRequestAction, {isOnline} = {}) {
+    createIOUReportAction(
+        'cancel',
+        moneyRequestAction.originalMessage.amount,
+        moneyRequestAction.originalMessage.currency,
+        {
+            IOUTransactionID: moneyRequestAction.originalMessage.IOUTransactionID,
+            isOnline,
+        },
+    );
+}
+
+beforeEach(() => {
+    reportActions = [];
+    const chatReportID = ReportUtils.generateReportID();
+    const amount = 1000;
+    const currency = 'USD';
+
+    // Create an IOU report
+    iouReport = ReportUtils.buildOptimisticIOUReport(
+        ownerEmail,
+        managerEmail,
+        amount,
+        chatReportID,
+        currency,
+        'en',
+    );
+
+    // The starting point of all tests is the IOUReport containing a single non-pending transaction in USD
+    // All requests in the tests are assumed to be offline, unless isOnline is specified
+    createIOUReportAction('create', amount, currency, {IOUTransactionID: '', isOnline: true});
+});
+
+describe('isIOUReportPendingCurrencyConversion', () => {
+    test('Requesting money offline in a different currency shows the IOUReport as pending', () => {
```

Comment:
The test description doesn't seem consistent with other descriptions, please try to keep it consistent.

```
Requesting money offline in a different currency will show the pending conversion message
```